CdnEngine_Azure_MI_Utility.php

```diff
@@ -0,0 +1,352 @@
+<?php
+
+namespace W3TC;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	die;
+}
+
+/**
+ * 
+ * This class defines utility functions for Azure blob storage access using Managed Identity.
+ * @author  Zubair <zmohammed@microsoft.com>
+ */
+class CdnEngine_Azure_MI_Utility {
+
+    const ENTRA_API_VERSION = '2019-08-01';
```

Comment:
these API versions are latest recommended versions?

Reply:
I see that as the only value in the [API documentation](https://learn.microsoft.com/en-us/azure/app-service/overview-managed-identity?tabs=portal%2Chttp#rest-endpoint-reference).